### PR TITLE
[Merged by Bors] - feat: port AlgebraicGeometry.Stalks

### PR DESCRIPTION
**Pull request by jjaassoonn**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/Stalks.lean [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+open AlgebraicGeometry
+
+open TopologicalSpace
+
+open Opposite
+
+variable {C : Type u} [Category.{v} C] [HasColimits C]
+
+attribute [local aesop safe cases (rule_sets [CategoryTheory])] Opens
```
Comment (semorrison) [inline Mathlib/AlgebraicGeometry/Stalks.lean:40]: Oh, but you deleted the commented out `op_induction`! We'll want to restore that functionality, too.

Comment (jjaassoonn) [reply Mathlib/AlgebraicGeometry/Stalks.lean:40]: Ah yes, let me restore it

Comment (jcommelin) [reply Mathlib/AlgebraicGeometry/Stalks.lean:42]: Created an issue for this: #4551

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**